packages/amplify-opensearch-simulator/package.json

```diff
@@ -17,7 +17,7 @@
   ],
```

Comment:
because this is a scoped package, it needs
```
  "publishConfig": {
    "access": "public"
  },
```
added to the `package.json` to prevent errors when publishing to NPM